apps/dotcom/sync-worker/src/TLDrawDurableObject.ts

```diff
@@ -382,6 +395,12 @@ export class TLDrawDurableObject extends DurableObject {
 				if (file.isDeleted) {
 					return closeSocket(TLSyncErrorCloseEventReason.NOT_FOUND)
 				}
+
+				const testFile = await isTestFile(this.env, file.id)
+				if (testFile && !(await canAccessTestProductionFile(this.env, auth))) {
+					return closeSocket(TLSyncErrorCloseEventReason.NOT_FOUND)
+				}
```

Comment:
should there be an `else if (testFile) { also fail }` clause here? I guess if it has `shared: false` it's no biggie

Reply:
Think our regular check a bit further down should cover that case?